dashboard/app/views/congrats/_congrats.html.haml

```diff
@@ -0,0 +1,9 @@
+- content_for(:head) do
+  %script{src: minifiable_asset_path('js/congrats/_congrats.js')}
```

Comment:
If we move this script tag to the bottom of the page we don't have to wait for `$(document).ready(showContent);` — we can call ReactDOM.render right away.